src/kernel/arch/x86/gdt.zig

```diff
@@ -9,58 +9,56 @@ const TABLE_SIZE: u16 = @sizeOf(GdtEntry) * NUMBER_OF_ENTRIES - 1;
 // The indexes into the GDT where each segment resides.
 
 /// The index of the NULL GDT entry.
-const NULL_INDEX: u16           = 0x00;
+const NULL_INDEX: u16 = 0x00;
```

Comment:
We have yet to decide on a style guide so could omit these formatting changes until we do big old clean up and go on a formatting spree. @DrDeano ?

Reply:
Scratch that, we've decided to follow zig fmt so we'll keep these changes.